dokku

```diff
@@ -199,6 +203,10 @@ case "$1" in
           docker stop $oldid \
           || docker kill $oldid \
           || plugn trigger retire-container-failed $APP # plugin trigger for event logging
+
+          # Attempt to remove the old container regardless of whether stopping/killing succeeded
+          docker rm $oldid \
```

Comment:
This also breaks the troubleshooting chain. See my [comments](https://github.com/progrium/dokku/issues/1505#issuecomment-148729833)